ITEMS.md

```diff
@@ -0,0 +1,147 @@
+# IOTMs and Useful Items used by Garbo
+
+In an ideal world, Garbo is supposed to be a flexible script that operates successfully for users regardless of their existing IOTM loadout. While Garbo has -support- for quite a lot of IOTMs, it doesn't really -require- any of them -- Garbo will do its best to make you as much meat as possible given the resources available to you. All that said, there are a few (relatively) cheap items that Garbo uses that can be useful to pick up. While Garbo does buy things like your diet & your potions, these are items where Garbo will not purchase them itself (as they don't pay for themselves in one distinct session of Garbo), but do make Garbo better enough to be worth looking into acquiring.
+
+| Item                              | Type      | Notes                                                                                                |
+| --------------------------------- | --------- | ---------------------------------------------------------------------------------------------------- |
+| mafia pointer finger ring         | Accessory | Gain meat on critical hits. Bad for embezzlers, but generally worn (almost) all day in Barf.         |
+| mafia thumb ring                  | Accessory | Gives an adventure on turn-taking combats 4% of the time. One of the best accessories.               |
+| lucky gold ring                   | Accessory | Piraterealm reward. Extra funfunds & Freddies; very high MPA accessory, much higher than you'd think |
+| belt of loathing                  | Accessory | +10 familiar weight; fantastic Barf accessory if you don't have the above.                           |
+| Stooper                           | Familiar  | Liver +1, ergo, quite a few extra turns each day                                                     |
+| Gingerbread Mob Hit               | Skill     | Insta-kill (1x a day)                                                                                |
```

Comment:
I think insta-kill should be renamed to free kill. Since insta-kills and free kills are distinct concepts.